custom_components/tuya_local/devices/inkbird_int12bw_thermometer.yaml

```diff
@@ -0,0 +1,1230 @@
+name: BBQ thermometer
+products:
+  - id: lkrzzdaex96syshadps
+    model: INT-12-BW
+    manufacturer: Inkbird
+
+# Probe1 is the black probe
+# Probe2 is the white probe
+
+entities:
+  # Base unit
+
+  - entity: sensor
+    class: battery
+    dps:
+      - id: 103
+        type: base64
+        name: sensor
+        mask: "000000FF"
+        endianness: little
+        unit: "%"
+
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 101
+        type: string
+        name: option
+        mapping:
+          - dps_val: C
+            value: celsius
+          - dps_val: F
+            value: fahrenheit
+
+  # r/w in query things but doesn't turn off the base
+  # - entity: binary_sensor
+  #   name: Base Power
+  #   class: power
+  #   category: diagnostic
+  #   dps:
+  #     - id: 102
+  #       type: boolean
+  #       name: sensor
+
+  - entity: number
+    name: Screem brightness
+    category: config
+    dps:
+      - id: 104
+        type: integer
+        name: value
+        unit: "%"
+        range:
+          min: 1
+          max: 100
+
+  - entity: select
+    name: Alarm volume
+    category: config
+    dps:
+      - id: 106
+        type: string
+        name: option
+        mapping:
+          - dps_val: "EQJM"
+            value: "mute"
+          - dps_val: "WgGJ"
+            value: "low"
+          - dps_val: "WgKA"
+            value: "medium"
+          - dps_val: "WgOH"
+            value: "high"
+
+  - entity: select
+    name: Automatic sleep
```

Comment:
Add `translation_key: timer` to get the correct translations for the options below.